### PR TITLE
workaround for sysinfo build failures on trueNAS freebsd 12 system

### PR DESCRIPTION
**Pull request by up2geek**
I'm not sure how well this will work elsewhere, but it appears to have worked for me.

### CODE REVIEW

**Comment by up2geek** [comment]
Ah nice. it appears your build-and-test pipeline has identified a few more areas that need updating... 

I might be completely out of my depth here, but I'm going to attempt it for my own installation at the very least.

**Comment by up2geek** [comment]
Just to be clear I have no experience with rust or this project before today, so I fully admit that I may have missed something major here, but I'm so glad to have this working on my TrueNAS/freebsd system and hope it helps others.

I have done small tests with duplicate test files, but I'm about to fire off big job and see how the performance compares to fdupes and others.

**Comment by up2geek** [comment]
@pkolaczk - ah... looks like there is a 'is_removable' function in the new version... I'm not concerned with removable drives at the moment so the current changes worked in the build I'm using at the moment, but this would ensure it stays compatible.
https://docs.rs/sysinfo/0.23.10/sysinfo/trait.DiskExt.html#tymethod.is_removable

Just want to say thanks for this tool, I've already cleaned up close to a TB of data using it. It's working well 👍

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0422/121950.949:ERROR:registration_protocol_win.cc(102)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (pkolaczk) [inline debug.log:1]: This file shouldn't be committed.

**Review by pkolaczk — APPROVED** [review]
Looks good. Thank you! 
Please remove that one accidentally committed file and I'll merge.

**Comment by pkolaczk** [comment]
Merged with minor cleanup in #118.

**Pull request closed without merging**